ci/playbooks/multinode-autohold.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: "Run ci/playbooks/multinode-autohold.yml"
-  hosts: controller
+  hosts: "{{ cifmw_zuul_target_host | default('all') }}"
```

Comment:
Note: The default value changes to align this playbook with the general approach we have with all the playbooks in the ci directory that takes the `cifmw_zuul_target_host`.